src/main/java/teammates/ui/webapi/action/SearchInstructorsAction.java

```diff
@@ -26,13 +29,34 @@ public void checkSpecificAccessControl() {
         }
     }
 
+    private void addAdditionalSearchFields(InstructorsData instructorsData, List<InstructorAttributes> instructors) {
+        instructorsData.getInstructors()
+                .forEach((InstructorData data) -> {
+                    AccountAttributes account = logic.getAccount(data.getGoogleId());
+                    if (account != null) {
+                        String institute = StringHelper.isEmpty(account.institute) ? "None" : account.institute;
+                        data.setInstitute(institute);
+                    }
+
+                    // Add registration key
```

Comment:
I think you should put the logic inside the construction of `InstructorData`.

Reply:
That makes sense. Should I do the same for `StudentData`? I didnt do it because `StudentData` is also returned to student users, and future developers will have to remember to set the key to null if they use `StudentData` in the future.

Reply:
fixed. thanks!